posthog/event_usage.py

```diff
@@ -144,17 +162,48 @@ def report_bulk_invited(
             "current_member_count": current_member_count,
             "email_available": email_available,
         },
+        groups=groups(user.current_organization, user.current_team),
     )
 
 
-def report_org_usage(distinct_id: str, properties: Dict[str, Any]) -> None:
+def report_org_usage(organization_id: str, distinct_id: str, properties: Dict[str, Any]) -> None:
     """
     Triggered daily by Celery scheduler.
     """
     posthoganalytics.capture(
-        distinct_id, "organization usage report", properties,
+        distinct_id,
+        "organization usage report",
+        properties,
+        groups={"organization": organization_id, "instance": SITE_URL},
+    )
+    posthoganalytics.group_identify("organization", organization_id, properties)
+
+
+def report_org_usage_failure(organization_id: str, distinct_id: str, err: str) -> None:
+    posthoganalytics.capture(
+        distinct_id,
+        "organization usage report failure",
+        properties={"error": err,},
+        groups={"organization": organization_id, "instance": SITE_URL},
+    )
+
+
+def report_user_action(user: User, event: str, properties: Dict = {}):
```

Comment:
Nit: naming convention. This seems to be a wrapper for any capture call we run, not necessarily triggered from a user action.

Reply:
All of the call sites are user triggered though - it wouldn't be appropriate to use this in a cronjob either (since you wouldn't know the current active project/org)